far/FarEng.hlf.m4

```diff
@@ -4225,54 +4225,54 @@ and the '|' character separates include masks from exclude masks.
 
 @SelectFiles
 $ #Selecting files#
-    Для обработки файлов и папок ~панели файлов~@FilePanel@ они могут быть
-помечены (будет обработана помеченная группа). Если ни один элемент панели не
-помечен, то будет обработан только элемент под курсором.
+    ~File panel~@FilePanel@ items (files and folders) can be selected
+for group processing. If no items are selected, only the item under
+cursor will be processed.
 
- #Пометка с клавиатуры#
+  #Keyboard Selection#
 
-    #Ins# помечает или снимает пометку с файла или папки под курсором и перемещает курсор вниз,
-#Shift+Клавиши курсора# позволяют перемещать курсор в различных направлениях.
+    #Ins# toggles selection on the item under cursor and moves the
+cursor down.
 
-    #Gray +# и #Gray -# выбирают и снимают пометку с группы с использованием
-одной или нескольких разделённых запятыми ~масок файлов~@FileMasks@. #Gray *#
-инвертирует текущую пометку. Если параметр #Пометка папок# в диалоге ~Настроек панели~@PanelSettings@
-включен, то пометка меняется и для папок.
+    #Shift+Arrow keys# move the cursor while selecting or deselecting
+items along the way. The action (selection or deselection) depends
+on the state of the item under cursor before pressing the key
+combination.
 
-    #Ctrl+<Gray +># и #Ctrl+<Gray -># выбирают и снимают пометку со всех
-файлов с тем же расширением, что и у файла под курсором. Если параметр
-#Пометка папок# в диалоге ~Настроек панели~@PanelSettings@ включен, то
-пометка меняется и для папок.
+    #Gray +# selects, and #Gray -# deselects the items using one or more
+~File masks~@FileMasks@. #†#
```

Comment:
@MKadaner , what exactly these `†` for? (just noticed)

Reply:
I meant a footnote which applies to several paragraphs. The footnote text is right below the `Shift+<Gray +>` paragraph. I tried star, like this `*)`, but it did not look right. The dagger symbol looks better. I did not like the way it was done in the original version and wanted to avoid the repetition.

If you can suggest better approach, I'll be happy to fix it.

Reply:
Ah, footnote, right.
It's just a little confusing because five sequential paragraphs refer to the same footnote and the footnote is not actually in the page footer :)